app/services/utils/raster.py

```diff
@@ -1,34 +1,75 @@
+import base64
+import io
+
+from PIL import Image
```

Comment:
I know the PR is already merged, but I'm having an error with this dependency. @denarch22 please create a new branch to include this dependency in the requirements file